pkg/sink/kafka/admin.go

```diff
@@ -42,8 +43,8 @@ type saramaAdminClient struct {
 }
 
 const (
-	defaultRetryBackoff  = 20
-	defaultRetryMaxTries = 3
+	defaultRetryBackoff     = 20
+	defaultRetryMaxDuration = time.Duration(120) * time.Second
```

Comment:
Is 120 seconds too long?

Reply:
Restarting a network switch or restarting kafka might take some time. Why would retrying for 2m be too long?